src/frontend/web_application/webpack/webpack.config.browser.js

```diff
@@ -10,6 +10,7 @@ const isProd = process.env.NODE_ENV === 'production';
 
 let config = Object.assign(baseConfig.getBase('browser'), {
   entry: [
+    'babel-polyfill',
```

Comment:
So, `babel-polyfill` is the polyfill direction the project is taking?

https://github.com/CaliOpen/Caliopen/issues/215